### PR TITLE
Beets 472 Correct Inoperable Assets funcitons

### PR DESCRIPTION
**Pull request by levertyk**
### Definition of Done:

* [x] Have you checked your [Jira](https://bee-cs458.atlassian.net/jira/software/projects/BEETS/boards/2/backlog) task, to make sure the user acceptance criteria are met?
* [x] Does your code have comments describing its function and reason?

### Source Control

* [x] Is all relevant code checked-in?
* [x] Is this branch ready to be reviewed by the Tech Lead?

### Formatting

* [x] Do your variable names make sense (to others)?
* [x] Have you removed unnecessary Console.logs? 
* [x] Is your code style consistent with other code in the file or the [Style Guide](https://google.github.io/styleguide/jsguide.html)

Screenshot 2023-04-18 at 3 59 31 PM
Screenshot 2023-04-18 at 3 59 34 PM

### CODE REVIEW

**Comment by piratecody** [comment]
image

I do not know if this is intended, but you can still checkout inoperable assets by entering the asset tag

File: client/src/components/AssetTable/AssetRow/AssetRow.js [hunk 1]
```diff
@@ -105,47 +109,50 @@ function AssetRow(props) {
...
       <td>{asset.description}</td>
       <td>{formattedDate}</td>
-      <td>
-        {
-          categoryLabel
-        }
-      </td>
-      <td>{asset.checked_out ? "No" : "Yes"}</td>
+      <td>{categoryLabel}</td>
+      <td>{available ? "No" : "Yes"}</td>
```
Comment (piratecody) [inline client/src/components/AssetTable/AssetRow/AssetRow.js:155]: This might look better as the icons used in the Single Asset Pages

**Review by piratecody — CHANGES_REQUESTED** [review]

File: client/src/components/AssetTable/AssetRow/AssetRow.js (hunk 1 above)
Comment (levertyk) [reply client/src/components/AssetTable/AssetRow/AssetRow.js:155]: Good idea, I can implement that as above and beyond the DOD.

**Comment by levertyk** [comment]
> image
> 
> I do not know if this is intended, but you can still checkout inoperable assets by entering the asset tag

Screenshot 2023-04-19 at 2 03 07 PM

**Review by piratecody — APPROVED** [review]

**Pull request merged**